README.md

```diff
@@ -23,7 +19,31 @@ to redefine the user agent via the `Object.defineProperty` method after the impo
 - **Main**
 
 ```ts
-// todo: @hugo update
+import ECx from 'electron-chrome-extension';
+
+app.on('ready', async () => {
+  createWindow();
+
+  // Load Grammarly
+  await ECx.load('kbfnbcaeplbcioakkpcpgfkobkghlhen');
+});
+```
+
+### Connect your application logic
+
+Extensions react to browser events for trigger their own logic.
+ECx send inner webContents events to extensions subscribers but external events
+like creating new window, changing tab focus etc should be triggered on your side like this:
+
+```js
+// main.js
+
+ipcMain.on('YOUR_EVENT', (...) =>
+  ECx.sendEvent({
```

Comment:
I guess you introduced this API specifically to transfer the tab focus/unfocus to ECX, right?
I would have preferred a more specific API in this case, that more explicit (here user needs to guess that `channel` to use) and that does not depend on underlying communications mechanism (event bus).

Something more like:
```ts
ecx.setTabsProvider(provider: TabsProvider);
interface TabsProvider {
  on('activated', {tabId, windowId }),
  // in future TabsProvider can have more APIs like
 // `getTabs` etc..
}
```

Reply:
Need to think about this with the next implementations